README.md

```diff
@@ -96,4 +96,13 @@ https://user-images.githubusercontent.com/2389735/180101651-86ba2084-4196-494b-9
   - [ ] add shadow bias material property to instances
 - [ ] add adaptive exposure based on histogram
 - [ ] make sure the skybox rad texture resolution is capped at a sane level; it causes the renderer to break on my m1.
-- [ ] use limit constraints at device creation time to try to lower the min_storage_buffer_offset_alignment number cuz smaller buffer = more cache hits
\ No newline at end of file
+- [ ] use limit constraints at device creation time to try to lower the min_storage_buffer_offset_alignment number cuz smaller buffer = more cache hits
+
+## Profiling
+You can profile wgpu-sandbox with [tracy](https://github.com/wolfpld/tracy) via [profiling](https://github.com/aclysma/profiling) crate.
+To do that you have to:
+- Download [tracy 0.9](https://github.com/wolfpld/tracy/releases/tag/v0.9)
+- Build wgpu-sandbox by adding --features="profile-with-tracy" to the cargo command
```

Comment:
It's about time this project got a real name lol, `wgpu-sandbox` is missing some character. I think I'll rename it to `ikari` for no particular reason, just sounds cool :)

Reply:
ikari sounds good to me!